ds/ds.php

```diff
@@ -2463,6 +2479,15 @@ public function isEmpty(): bool {}
          */
         public function peek() {}
 
+        /**
+         * Removes and returns the value with the highest priority
+         * @link https://www.php.net/manual/en/ds-priorityqueue.pop.php
+         *
+         * @return mixed The removed value which was at the front of the queue.
+         * @throws UnderflowException if empty.
+         */
+        public function pop() {}
+        
```

Comment:
Please run php-cs-fixer as described in readme to reformat the code. Or just remove whitespaces here, per failed tests: https://github.com/JetBrains/phpstorm-stubs/pull/1430/checks#step:5:8